publish.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm -rf ./output
+git clone https://github.com/OpenRA/openra.github.io.git output
```

Comment:
This would be nicer it it were the ssh URL: `git@github.com:openra/openra.github.io`. Although explicitly entering your username and password may keep you from breaking stuff.